client/components/notice/index.jsx

```diff
@@ -54,7 +54,13 @@ export const Notice = React.createClass( {
 		let content, text;
 
 		if ( typeof this.props.children === 'string' ) {
-			return <span className="notice__text"><span>{ this.props.children }</span></span>;
+			return (
+				<span className="notice__content">
+					<span className="notice__text">
+						<span>{ this.props.children }</span>
```

Comment:
So the span was added as part of the React 15 upgrade (c2101acf7338e497e083cc4bfa0aa5e6cf7a7b5f) and corresponds to this style:

https://github.com/Automattic/wp-calypso/blob/5da3f14/client/components/notice/style.scss#L74-L77

But the `max-width` doesn't seem to work anyways, I suspect because `span` is an inline element. Instead, if we want to enforce a `max-width` on the notice text, I suggest we do so on the parent `.notice__text` class and drop the spans.

I'll work on this, but adding a comment since I'm guessing this width enforcement doesn't really apply (at least consistently) in master currently.